mysql/README.md

```diff
@@ -1,59 +1,131 @@
 # Overview
 
-Connect MySQL to Datadog in order to:
+The Datadog agent's MySQL check sends many database metrics to Datadog, including metrics for:
 
-* Visualize your database performance
-* Correlate the performance of MySQL with the rest of your applications
+* Query throughput
+* Query performance (average query run time, slow queries, etc)
+* Connections (currently open connections, aborted connections, errors, etc)
+* InnoDB (buffer pool metrics, etc)
 
-# Installation
+And [many more](https://github.com/DataDog/integrations-core/blob/master/mysql/metadata.csv). You can also invent your own metrics using custom queries.
 
-1. Create a ```datadog``` user with replication rights in your MySQL server:
+The MySQL check sends one service check: whether or not the Datadog agent can connect to MySQL.
 
-        sudo mysql -e "CREATE USER 'datadog'@'localhost' IDENTIFIED BY '<UNIQUEPASSWORD>';"
-        sudo mysql -e "GRANT REPLICATION CLIENT ON *.* TO 'datadog'@'localhost' WITH MAX_USER_CONNECTIONS 5;"
-2. Verify that the above commands worked by running:
+It does not send anything to your events stream.
```

Comment:
Yeah, I also noticed the ambiguity in the word 'events' here. I initially had something like "It does not send any events to Datadog", which is even worse, and I thought I'd fixed it with what's in the PR now. But you still recognized the ambiguity - darn. I'm still learning all the Datadog nomenclature - I thought 'events stream' would make things clear enough but perhaps not.

I like the brevity of your suggestion, but in its brevity it could also sound ambiguous to some users. Emit to what, exactly?

To take your suggestion of explaining fully, how about: 

"It does not send any events to Datadog; metrics that are related to 'events' in MySQL - e.g. ` `mysql.innodb.mutex_os_waits` - will be submitted *as metrics*,  but the MySQL check does not send any events to your [events stream](https://app.datadoghq.com/event/stream)."

Or, what is the meaning of the word "events" with respect to MySQL that makes the original wording most ambiguous? 

One final option: to remove any mention of events in Overview (and omit any mention of service checks in integrations that have no service checks, etc). However I noticed that in app.datadog, there's a tickbox for MySQL in the lefthand menu on the events stream page, so, it feels useful to hint *somewhere* that using this filter will always turn up an empty events stream.